File: aura-test-workspace/sample.ts
// Sample TypeScript file for testing Aura AI features
interface User {
    id: number;
    name: string;
    email: string;
}

class UserService {
    private users: User[] = [];
    
    async createUser(userData: Partial<User>): Promise<User> {
        // TODO: Add validation
        const newUser: User = {
            id: Date.now(),
            name: userData.name || '',
            email: userData.email || ''
        };
        this.users.push(newUser);
        return newUser;
    }
    
    findUserById(id: number): User | undefined {
        return this.users.find(user => user.id === id);
    }
}

export { UserService, User };

File: aura-test-workspace/sample.py
# Sample Python file for testing Aura AI features
class Calculator:
    def __init__(self):
        self.history = []
    
    def add(self, a, b):
        result = a + b
        self.history.append(f"{a} + {b} = {result}")
        return result
    
    def multiply(self, a, b):
        result = a * b
        self.history.append(f"{a} * {b} = {result}")
        return result
    
    def get_history(self):
        return self.history

if __name__ == "__main__":
    calc = Calculator()
    print(calc.add(5, 3))
    print(calc.multiply(4, 7))
